File: classexercise/java/Investment.java
import java.util.Scanner;
public class Investment{
public static void main(String...args){

	int totalAmount = 0;
	int originalAmount = 1000;
	int annualRate = 7 / 100;
	for(int year = 1; year <= 30; year++){
	double principalOfTheYear = originalAmount * (Math.pow((1 + annualRate), year));
	totalAmount += principalOfTheYear;
	System.out.printf("\nThe amount for year %d is %d", year, totalAmount);
	}
	System.out.printf("\nThe total amount for 30years", totalAmount);


}
}

File: assignments/chapter3/investment.py
totalamount = 0
original_amount = 1000
annual_rate = 7 / 100
for years in range(1, 31):
	principal_of_theyear = original_amount * ((1 + annual_rate) ** years)
	totalamount += principal_of_theyear
	print('The amount for year', years, 'is', totalamount)
print('The total amount for 30 years is ', totalamount)








